common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -530,8 +535,8 @@
                         } else {
                             if (state.isTouch) {
                                 self.subtitlesEl.find('.subtitles-menu')
-                                    .text(gettext('Transcript will be displayed when you start playing the video.')) // jshint ignore: line
-                                        .wrapInner('<li></li>');
+                                    .text(window.gettext('Transcript will be displayed when you start playing the video.')) // jshint ignore: line
```

Comment:
Hm, I thought I changed all of those back. I'm going to remove the `windows` in this use.